src/NumericalAlgorithms/LinearOperators/DefiniteIntegral.hpp

```diff
@@ -16,15 +16,22 @@ class Mesh;
 
 /*!
  * \ingroup NumericalAlgorithmsGroup
- * \brief Compute the definite integral of a grid-function over a manifold.
+ * \brief Compute the definite integral of a grid function over a manifold.
  *
- * The integral is computed on the reference element by multiplying the
- * DataVector with the Spectral::quadrature_weights() in that
- * dimension.
- * \requires number of points in `integrand` and `mesh` are equal.
- * \param integrand the grid function to integrate.
- * \param mesh the Mesh of the manifold on which `integrand` is located.
- * \returns the definite integral of `integrand` on the manifold.
+ * Given a function \f$f\f$, compute its integral \f$I\f$ with respect to
+ * the reference coordinates \f$(\xi, \eta, \zeta)\f$. In 1 dimension,
```

Comment:
Multiple times:  we use `logical coordinates` instead of `reference coordinates` throughout the code...